zerver/tests/test_email_notifications.py

```diff
@@ -1515,3 +1542,23 @@ def test_followup_day2_email_delay(self) -> None:
         # Test date_joined == Friday
         user_profile.date_joined = datetime(2018, 1, 5, 1, 0, 0, 0, tzinfo=timezone.utc)
         self.assertEqual(followup_day2_email_delay(user_profile), timedelta(days=3, hours=-1))
+
+
+class TestCustomEmailSender(ZulipTestCase):
+    name = "Right Said Fred"
+    email = "ur@toosexy.com"
```

Comment:
Let's use the bundle of `self.nonreg_email()` functions to get an email address for this -- we try hard to avoid having potentially real email addresses outside the `@example.com` and `@zulip.com` domains in the codebase, and to keep our example users/emails in the development environment's general Shakespeare theme.

(We have a stalled goal of replacing all the `@zulip.com` in the development environment with `@example.com`, which is why we have these helpers like `self.example_user("hamlet")` to avoid copy-pasting their account details everywhere).

Reply:
DONE: moved to [PR 22060](https://github.com/zulip/zulip/pull/22060) incl replacing a bunch of @zulip.com's